support/build/extensions/no-debug-non-zts-20121212/sundown

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+dep_name=$(basename $BASH_SOURCE)
+
+source $(dirname $BASH_SOURCE)/../pecl
```

Comment:
well, it is a bit ugly, but your copy needs to have similar ugly paths too, so the copy does not even save us from this (and makes maintenance harder)